cms/templates/certificates.html

```diff
@@ -115,6 +118,9 @@ <h2 class="title-3">${_("Other Course Settings")}</h2>
               <li class="nav-item"><a href="${course_team_url}">${_("Course Team")}</a></li>
               <li class="nav-item"><a href="${advanced_settings_url}">${_("Advanced Settings")}</a></li>
               <li class="nav-item"><a href="${utils.reverse_course_url('group_configurations_list_handler', context_course.id)}">${_("Group Configurations")}</a></li>
+              % if exams_url:
+              <li class="nav-item"><a href="${exams_url}">${_("Exam Settings")}</a></li>
```

Comment:
Does this need to be indented?

Reply:
I've seen it both indented and not indented in the repo, is there a preference? For the purpose of readability I agree that it should be indented. What are your thoughts?

Reply:
My first hunch would be indented, but I also am not that familiar with mako templates and whether it matters. If the page looks the same regardless of the indentation and you've seen both styles out there, then I would indent it. Like you, I think it would improve readability :).